docs/concepts/package-types.md

```diff
@@ -0,0 +1,225 @@
+are compiled as [ES
+Modules(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)]
+and all their external `import`s are rewritten to point to an
+[ESM CDN](../esm-views/esm-cdn.md). Like apps, they are be built with the same
+Webpack or esbuild configuration and support the same functionalities. Unlike
+apps, they don't allow the user to specify a custom `index.html`, they don't
+include a `public` folder in the build and they expect their entrypoint
+(`src/index.tsx`) to not `render` to the DOM, but to export something, typically
+a React Component.
+
+Esm Views are Modular's type of choice to implement the
+[micro-frontend](../concepts/microfrontends.md) pattern. They typically expect
+their entrypoint exports to be
+[dynamically `import`ed at runtime](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import)
+by some other micro-frontend, but can be also
+[served standalone](../esm-views/how-to-build.md) exactly as an `app`, provided
+that they default export a React component that can be rendered to the DOM.
+
+Since the micro-frontend pattern allows different teams to build and serve their
+applications independently and compose them at runtime (parallelizing otherwise
+expensive build operations), third-party dependencies need to be de-duplicated
+across micro-frontends; for this reason, external dependencies are not bundled,
+but they are rewritten to a [configurable ESM CDN](../esm-views/esm-cdn.md) that
+serves them as ES modules on the fly.
+
+Esm Views can be built using [`modular build`](../commands/build.md): the
+resulting output is an optimized site that can be either served statically
+(using a synthetically-generated `index.html` file and a `_trampoline.js`
+loader), or dynamically imported by another ESM view. The js and CSS
+entrypoints, in the second case, are linked in the
+[generated manifest file](../esm-views/output-package-manifest.md), that the
+loading code can fetch and examine before loading the `esm-view`. When building
+an ESM View, all "local" code (files in `src`) is bundled in a single blob that
+can be split into different files. All external dependencies are rewritten to an
+external CDN by default and don't get bundled, although this behavior is
+[configurable](../configuration.md).
+
+ESM Views are generated by `modular add` using the
+[`modular-template-esm-view`](https://github.com/jpmorganchase/modular/tree/main/packages/modular-template-esm-view)
+[template](../concepts/templates.md)
+
+## Package
+
+Modular `package`s are generic, publishable libraries with a single entrypoint
+file. They are built with [Rollup.js](https://rollupjs.org/guide/en/) and they
+are not bundled in a single blob; files required directly or indirectly from the
+entry point are separately transpiled as CJS Modules and ES Modules (althought
+it's not a 1:1 correspondance: the compilation process typically creates
```

Comment:
```suggestion
it's not a 1:1 correspondence: the compilation process typically creates
```

Reply:
I _think_ it's supposed to be spelled that way. Not sure if US/UK thing

Reply:
"Correspondance" is the French word, so you're right :)